models/public/efficientdet-d0-tf/efficientdet-d0-tf.md

```diff
@@ -0,0 +1,86 @@
+# efficientdet-d0-tf
+
+## Use Case and High-Level Description
+
+The "efficientdet-d0" model is one of the [EfficientDet](https://arxiv.org/abs/1911.09070)
+models  designed to perform object detection. This model was pretrained in TensorFlow*.
+All the EfficientDet models have been pretrained on the MSCOCO* image database.
+For details about this family of models, check out the Google AutoML [repository]
+(https://github.com/google/automl/tree/master/efficientdet).
+
+## Example
+
+## Specification
+
+| Metric            | Value           |
+|-------------------|-----------------|
+| Type              | Object detection|
+| GFLOPs            |     -           |
+| MParams           |     -           |
+| Source framework  | TensorFlow\*    |
+
+## Accuracy
+
+| Metric | Converted model |
+| ------ | --------------- |
+| coco_precision | 31.95%|
+
+## Performance
```

Comment:
we are removing empty sections from documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please do this as well